api/src/main/java/org/openmrs/util/VelocityExceptionHandler.java

```diff
@@ -40,7 +42,15 @@ public Object methodException(Class claz, String method, Exception e) throws Exc
 		}
 		
 		// keep the default behavior
-		throw e;
+		try {
+			throw e;
+		}
+		catch (Exception x) {
+			
+			x.printStackTrace();
+		}
+		return info;
```

Comment:
@gitcliff , what happens when you leave out this change?

Reply:
@Ruhanga  the velocityExceptionHandler will generate a compilation error since when it throws an exception it has to be caught in the try and catch brackets

Reply:
@gitcliff you may want to follow java conventions such as those found here https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-MoreConventions in the case where it is necessary to print anything, but the above change is not necessary. So you may have to refactor to eliminate the 
`try {
 }
 catch {
 }`
statements.

Reply:
@Ruhanga  i removed the try and catch blocks and refactored again